apps/verticalhome/js/more_apps_navigation.js

```diff
@@ -4,7 +4,7 @@
   var MoreAppsNavigation = {
     items: null,
     selectedElemIndex: 0,
-
+    personalizeMode: false,
```

Comment:
actualy i need personalizeMode in moreApps, because i must be sure that moreapps are also able to edit